tensorflow/python/kernel_tests/nn_ops/pooling_ops_3d_test.py

```diff
@@ -287,6 +287,55 @@ def DISABLED_testAvgPool3dEmptyOutTensor(self):
     values = self.evaluate(avg_pool_3d)
     self.assertEqual(values.shape, (30, 1, 1, 0, 17))
 
+  def _getJacobians(self,
```

Comment:
Why split off this function and manually compare numeric and theoretical gradienst instead of calling `gradient_checker.compute_gradient_error`, like what was done before?

Reply:
I split them up so I can compre the bf16 analytical gradient to the fp32 numerical gradient as done in [some existing tests](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/nn_ops/bias_op_base.py#L212-L216). Otherwise using just compute_gradient_error the test is too inaccurate to be useful.

Reply:
You are right, I missed that when reviewing it the first time.